### ISSUE TITLE
Sitewide - Skip navigation links

### ISSUE THREAD

**Issue by gggrant** [GCTC-NTGC/TalentCloud#665]
# Description
Skip nav anchor links to main content on all pages. Should be the first item when a user hits tab.

See behaviour here: https://digital.canada.ca/

**Comment by gobyrne** [comment]
As far as I can tell, this does function the same as digital.canada.ca ... but it's not the first thing I encounter when I hit tab on a page (on either site)

**Comment by gggrant** [comment]
Really? It should be. Working fine for me in `dev`.
